beta/src/components/MDX/CodeBlock/CodeBlock.tsx

```diff
@@ -109,7 +109,7 @@ const CodeBlock = React.forwardRef(
     );
   }
 );
-
+CodeBlock.displayName = 'CodeBlock'
```

Comment:
?

Reply:
<img width="571" alt="Screenshot 2021-11-12 at 10 14 04 PM" src="https://user-images.githubusercontent.com/32865581/141503092-d7e28514-41b8-44d9-a165-fe5cd77b7cdb.png">

Reply:
That seems like a bad rule to enable.

Reply:
That came in by default. Shall I disable it then?

Reply:
Where is it coming from? Which config?

Reply:
It is coming from `react/display-name` 🤔  I extended the `next` config and it seems to include `eslint-plugin-react`
https://nextjs.org/docs/basic-features/eslint

Reply:
I don't see `react-display-name` in the list of rules on that page.

Reply:
<img width="696" alt="Screenshot 2021-11-12 at 11 51 21 PM" src="https://user-images.githubusercontent.com/32865581/141515915-6fd5227a-dd89-477d-83cb-b5ac22d7f86a.png">
and `eslint-plugin-react` has display-name
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md

Reply:
Okay I see what's happening now. It's just because of forwardRef.

Try doing this:

```js
const Foo = forwardRef(function Foo() {
 ...
})
```

Reply:
Done updates the arrow function to function CodeBlock and removed the `CodeBlock.displayName`